src/core/gltf2importer/gltf2parser.cpp

```diff
@@ -124,6 +136,19 @@ void extractPositionViewDirAndUpVectorFromViewMatrix(const QMatrix4x4 viewMatrix
                          viewMatrix.row(2)[3]);
 }
 
+int addToJsonArray(QJsonObject &object, const QString &name, const QJsonObject &toInsert)
```

Comment:
there are already some "JSON-util" like functions here : https://github.com/KDAB/kuesa/blob/dev/src/core/gltf2exporter/gltf2utils_p.cpp

maybe they should all be put in some kind of generic json_utils.hpp ?

Reply:
put it into gltf2utils_p.cpp for now